Resources/ProxyProvider/DIRACCAProxyProvider.py

```diff
@@ -0,0 +1,254 @@
+""" ProxyProvider implementation for the proxy generation using local (DIRAC)
+    CA credentials
+"""
+
+import os
+import re
+import glob
+import shutil
+import tempfile
+import commands
+
+from DIRAC import gLogger, S_OK, S_ERROR
+from DIRAC.Resources.ProxyProvider.ProxyProvider import ProxyProvider
+from DIRAC.Core.Security.X509Chain import X509Chain  # pylint: disable=import-error
+from DIRAC.ConfigurationSystem.Client.Helpers import Registry
+
+__RCSID__ = "$Id$"
+
+
+userConf = """[ req ]
```

Comment:
Could you please put the openssl configuration template in a separate file, whose location would be taken from the CS ? This allows any VO to specify their own SSL config

Reply:
This template as default solution if no configuration file exist, to use configuration file need just to set it in provider section as "CAConfigFile" option

Reply:
Ah yes, I did not see that :)